resources/testharness.js

```diff
@@ -2028,6 +2056,7 @@ policies and contribution forms [3].
         this.set_status(this.TIMEOUT, "Test timed out");
 
         tests.started = true;
+        tests.current_test = this;
```

Comment:
Not that we can do much about this, but so I'm clear - if someone fails to `step()` something, we'll end up attributing asserts to the wrong test, correct?

Reply:
I've changed things to set the current test back to null. I'm not sure that actually handles the case of an assert outside a test in a remote test, but honestly the test is broken either way.